README.md

```diff
@@ -9,6 +9,15 @@ C API to Qt for Qlift
 - `ENABLE_SHARED`: Enable or disable building qlift-c-api as a shared library. Default is ON.
 - `ENABLE_STATIC`: Enable or disable building qlift-c-api as a static library. Default is OFF.
 
+To install as a shared library in ```/usr/local```:
+```sh
+git clone https://github.com/Longhanks/qlift-c-api
+cd qlift-c-api
+cmake CMAKE_INSTALL_PREFIX=/usr/local .
```

Comment:
I've seen people do that (build in a new directory), but when I've tried it, Cmake doesn't put anything there. I can redo those commands though...